### PR TITLE
recipe for trivy

### PR DESCRIPTION
**Pull request by mediocretech**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by mediocretech** [comment]
@conda-forge/help-go ready for review

File: recipes/trivy/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+
+build:
+  number: 0
+  script:
+    - go install ./cmd/trivy
+
+requirements:
+  build:
+    - {{ compiler('go') }}
```
Comment (xhochy) [inline recipes/trivy/meta.yaml:20]:
```suggestion
    - {{ compiler('go-nocgo') }}
```

**Review by xhochy — CHANGES_REQUESTED** [review]
Please add go-licenses as done in other go recipes and also copy the build command from there to enable cross-compiling.

File: recipes/trivy/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+source:
+  url: https://github.com/aquasecurity/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: dfe18dbe98ce4edecbe15a830840e914f8234d99ac8074f7992c315a1ce98392
+
+
+build:
+  number: 0
+  script:
+    - go install ./cmd/trivy
+    - go-licenses save ./cmd/trivy --save_path licenses --ignore github.com/csaf-poc/csaf_distribution --ignore modernc.org/mathutil
```
Comment (xhochy) [inline recipes/trivy/meta.yaml:17]: You would need to add the two ignored licenses manually to the recipe directory.

**Review by xhochy — CHANGES_REQUESTED** [review]

File: recipes/trivy/meta.yaml (hunk 2 above)
Comment (mediocretech) [reply recipes/trivy/meta.yaml]: Thanks @xhochy.  I'm not too familiar with this paradigm, let me know if further changes are needed.

**Review by xhochy — APPROVED** [review]
Licenses look good

**Pull request merged**